src/group/communication/app.js

```diff
@@ -1,9 +1,11 @@
 import {
 	MOCK_GROUP_COMMUNICATION,
-	MOCK_GROUP_COMMUNICATION_COMMENTS,
 	MOCK_GROUP_COMMUNICATION_FOLLOWERS,
 	MOCK_GROUP_COMMUNICATION_SUGGESTED_FOLLOWERS
 } from './api';
+import {
+	MOCK_COMMENTS_LIST,
+} from './../api';
```

Comment:
there is nothing in `src/group/`: https://github.com/meetup/meetup-web-mocks/tree/master/src/group